tekton/resources/nightly-release/base/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Is the idea to have one file per resource type?
We might need more service accounts in the future. Currently we one service account to run the release event listener and then to run the release, but we may want to split that our in future on one SA for the listener that has no access to secrets, one a different one to run the release that does have access to secrets.

Reply:
Right now it doesn't make a big difference, but splitting each resource into different files seems like it would make it easier to support base reuse across different kustomize configs in the future. e.g. have one multiple base configs that reuse the same EventListener, Template, but doesn't create its own SA.

I don't think how this is currently laid out limits any changes we would want to make for service account usage later.

Reply:
ok